packages/veto-frontend/src/layouts/MainLayout/MainLayout.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent, useState } from 'react'
+import { Flex } from '@veriphi/veto-ui'
+import { Router } from '@reach/router'
+import Dashboard from '../../pages/Dashboard'
+import Wallets from '../../pages/Wallets'
+import History from '../../pages/History'
+import Explorer from '../../pages/Explorer'
+import Monitoring from '../../pages/Monitoring'
+import SideMenu from 'components/molecules/SideMenu'
+
+export type MainLayoutLabel = 'Dashboard' | 'Wallets' | 'History' | 'Explorer' | 'Monitoring'
+export type MainLayoutRoute = '/dashboard' | '/wallets' | '/history' | '/explorer' | '/monitoring'
+
+type Props = {
+  path: string
+}
+
+const getActiveMenu = (): MainLayoutRoute => {
+  const currentPathName: string = window.location.pathname
+  if (currentPathName !== '/dashboard' || '/wallets' || '/history' || '/explorer' || '/monitoring') {
```

Comment:
Are you working with Python theses days :P  
That patter does not work in JS, you can do something similar with `Array.prototype.has()`

```typescript
const pathWhitelist = ['/dashboard', '/wallets', '/history', '/explorer', '/monitoring'];
if(pathWhitelist.has(currentPathName)){
  return currentPathName;
} else {
  return '/dashboard';
}


```

Reply:
I think he mentioned working with Python recently actually! :P

But unfortunately `.has` is a Map prototype. 
In your example @drawm we'd have to use  the equivalent Array prototype `.includes` 

```javascript

if(
   ['/dashboard', '/wallets', '/history', '/explorer', '/monitoring'].includes(currentPathName)
){
   return currentPathName;
} else {
   return '/dashboard';
}

```

Reply:
Oh right! I always mix up the two :P `.includes` is the way to go !